cl/cleanup/management/commands/cleanup_636_pacer_leading_zeros.py

```diff
@@ -11,7 +11,8 @@
     get_local_document_url_from_path,
 )
 from cl.search.models import RECAPDocument, Docket
-from cl.scrapers.tasks import get_page_count, extract_recap_pdf
+from cl.scrapers.tasks import extract_recap_pdf
```

Comment:
Let's just remove this whole file. I tire of our cruft.

Reply:
√ removed